mirrors.yml

```diff
@@ -1,15 +1,18 @@
 origin:
-    desc: AOSC main repository (Aperture Science Limited)
+    desc: AOSC main repository with Auto-Redirect
```

Comment:
why cap letters?

Reply:
No need for capitalised letters:

AOSC main repository (auto-redirect)

Reply:
cap problem resolved.